### ISSUE TITLE
Show error delete product on order page admin

### ISSUE THREAD

**Issue by ytilotti** [PrestaShop/PrestaShop#25298]
#### Describe the bug

The errors on action delete product in the admin page of an order were not showing. It's OK on update & add action.

#### Expected behavior

Show an error like this:
![image](https://user-images.githubusercontent.com/5848945/125063852-4451a780-e0b0-11eb-8689-54d2d2ccf1aa.png)

#### Steps to Reproduce

1. Create 3 hook `actionObjectOrderDetailDeleteBefore`, `actionObjectOrderDetailAddBefore`, `actionObjectOrderDetailUpdateBefore` (with ps_qualityassurance)
2. Add this code in the hooks (for example):
`throw new PrestaShop\PrestaShop\Core\Module\Exception\ModuleErrorException(("Error " . __FUNCTION__));`
3. Go to an order page on the admin
4. Click on delete a product:
![image](https://user-images.githubusercontent.com/5848945/125062758-086a1280-e0af-11eb-8bcf-2f9caa32f81d.png)
5. Click on update a product, change the quantity and submit. An error appear.

#### Additional information

* PrestaShop version: 1.7.7.5
* PHP version: 7.2

**Comment by khouloudbelguith** [comment]
Hi @ytilotti,

Thanks for your issue and the PR.
I don't have an exception when updating the quantity of the product in the BO > Orders > View Order page.
I attached a screen record
https://drive.google.com/file/d/1yz8hn2Cf8bm6uqX3BlMkcWh0iouocp_O/view?usp=sharing
I'm missing something?
Thanks!

**Comment by ytilotti** [comment]
The problem come from ps_qualityassurance, it alterate Exception.
```
try {
    $output = eval($payload['content']);
    $this->recordHookCall($hookName, $params, $output);
    return $output;
} catch (Throwable $e) {
    $output = $e->getMessage();
    $this->recordHookCall($hookName, $params, $output, true);

    echo $output;
}
```
So need to create a module for testing or need to fix ps_qualityassurance.

**Comment by khouloudbelguith** [comment]
@ytilotti, Could you please provide me a demo module to test your issue?

Thanks!

**Comment by PierreRambaud** [comment]
@khouloudbelguith COuld you test with this PR in the module Ps_qualityassurance I fix an issue where module doesn't trigger exceptions https://github.com/PrestaShop/ps_qualityassurance/pull/18/files

**Comment by khouloudbelguith** [comment]
Hi @PierreRambaud,

Thanks, It works :+1: 
@ytilotti, I manage to reproduce the issue PS1.7.7.5
https://drive.google.com/file/d/1bxDdGIKsJ1ckCGisE4dxCxOVzYNZrsgA/view?usp=sharing
I'll add this to the debug backlog.

But how can I check if it is a regression or not with PS1769?
I don't have any errors with PS1769 using the ps_qualityassurance.

Thanks!

**Comment by ytilotti** [comment]
Thanks @khouloudbelguith for your tests. It's due to the new version of the order page with the migration on Symfony since 1.7.7.

**Comment by Progi1984** [comment]
Fixed by @ytilotti in #25291
